lego-powered-up/src/tests.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(unused)]
+#[cfg(test)]
+
+mod tests {
```

Comment:
Put the cfg next to the mod so that it's clearly attached to it
```suggestion
#![allow(unused)]

#[cfg(test)]
mod tests {
```